cob_datapipeline/operators/solr_api/base_operator.py

```diff
@@ -37,6 +38,19 @@ def get_failed_reason(response):
     except (JSONDecodeError, KeyError):
         return f'{response.status_code}:{response.reason}'
 
+def _safety_check(skip_included):
+    """
+    Checks that if skip_included is set, then it does not include a blank
+    value.  This is an extra safety measure against the possiblity of meaning
+    to skip deleting something but not configuring it properly.
+    """
+    if skip_included:
+        for name in skip_included:
+            if name in ['None', None, '']:
+                raise SafetyCheckException("skip_included MUST NOT contain any blank values.")
```

Comment:
@sensei100 I've added this other safety measure, per your suggestion.  I think this satisfies the situation where by some mistake `skip_included` has a blank value.  I don't think we've done this for the catalog OAI harvest safety check measures (or maybe it's satisfied some other way). But, in that case the worst that can happen is that we try to index to the prod DB not that the prod DB gets deleted! 😱 

I do agree that perhaps in this case we should take these precautions even further.  For the OAI harvest I had proposed that we ping the server as the source of truth for prod collection/DB.  But, at the time @bibliotechy did not think that we needed to go that far so we didn't implement that safety check.

If we want to go that route I think we need to decide whether we want to implement a tul_cob HTTP endpoint with info we can use in our safety check (as I think @bibliotechy has suggested) or whether we should SSH into the/a tul_cob server to use the environment variables (there are pros and cons for either choice which we can go into during a standup/meeting).

Reply:
@dkinzer I still think an HTTP endpoint exposing the current SOLR collection is a good idea.

Reply:
I agree. In that case this PR needs to be blocked until that endpoint is ready. I'll update the status back to draft.

Reply:
We'll be using okcomputer for this bit.